spring-cloud-aws-cloudmap-config/src/main/java/org/springframework/cloud/aws/cloudmap/AwsCloudMapProperties.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2013-2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.aws.cloudmap;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties(AwsCloudMapProperties.CONFIG_PREFIX)
+
+public class AwsCloudMapProperties {
```

Comment:
I followed the same model that we have for parameter store, where we have a module for reading config and a starter project for beans and initiation. Is it ok if I keep it the same way? Or do you want me to move it here? Because this might require a lot more refactoring, so thought of checking with you before that.